packages/babel-types/src/definitions/experimental.js

```diff
@@ -61,23 +61,3 @@ defineType("ExportNamespaceSpecifier", {
     }
   }
 });
-
-defineType("RestProperty", {
```

Comment:
If other people write plugins that visit these types, this would keep the visitors working.